tests/integration/features/serverless_feature_test.go

```diff
@@ -115,6 +130,21 @@ var _ = Describe("Serverless feature", func() {
 			var knativeServingCrdObj *apiextensionsv1.CustomResourceDefinition
 
 			BeforeEach(func() {
+				// Create KNativeServing Subscription
+				knativeSubscription := &ofapiv1alpha1.Subscription{}
+				Expect(yaml.Unmarshal([]byte(knativeServingSubscription), knativeSubscription)).To(Succeed())
+
+				ns := newNamespace("openshift-serverless")
+				_, err := controllerutil.CreateOrUpdate(context.Background(), envTestClient, ns, func() error {
+					return nil
+				})
+				Expect(err).ToNot(HaveOccurred())
+
+				_, err = controllerutil.CreateOrUpdate(context.Background(), envTestClient, knativeSubscription, func() error {
+					return nil
+				})
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
Extract to its own func? I want to know **what** is set up as a test fixture, but I can jump to the func to see **how** later on :)

Bonus points for reusing it in the service mesh test case.

Reply:
done here : ) 
7fc3fa7